### ISSUE TITLE
Mission file from destroy_oil_transport script is well generated, but cannot be seen from inside DCS ( 1.5.7 )

### ISSUE THREAD

**Issue by Flupke25** [pydcs/dcs#7]
Hi again,

Yes I would like to experiment with this script, but contrary to other mission files generated from other scripts, the files generated by this script cannot be seen from inside DCS. They can be seen in the file system at the right location. Do you experiment the same problem ? Would you have any clue of what it could be ?

Thank you in advance,

Flupke

**Comment by rp-** [comment]
Hi!

did you use the git master version? or you running the release from the pip installer?

Just generated a mission and I can open it in the latest dcs 1.5.7

**Comment by Flupke25** [comment]
Hello,

Thanks for the very quick reply :)

I used the git master version. Is it the good one ? 
It is indeed very strange not to see the file in DCS. I cannot figure out why it is like this for me, but I will search, and I let you know.

In the future, if I have ideas, new code or fixes for the library, I let you know as well ;)

Flupke

**Comment by rp-** [comment]
If you can upload the .miz file somewhere i can have a look what is wrong
with it. It will not show up in dcs if there is a parsing problem.

**Comment by Flupke25** [comment]
Ok, I send you the file generated this evening with the associated command to generate it, when I am back home.

**Comment by Flupke25** [comment]
[oil_transport.miz.zip](https://github.com/pydcs/dcs/files/1483122/oil_transport.miz.zip)

Here it comes.
you have to rename it without the ".zip".
It was generated with python 3.5.2, with the command : "python destroy_oil_transport" without parameters and is not visible in DCS.
Hope it can helps.

**Comment by rp-** [comment]
I know whats wrong in your mission, but I can't reproduce it on my end.
Are you sure you are on the latest master revision?
if you run git log the latest should be 57ba8210506b2b10906d63ede705692ed7a4f64b right now.

**Comment by Flupke25** [comment]
I have cloned this evening to be sure, but I have the same problem, I cannot see the generated file oil_transport.miz in DCS.
By the way, I had to put in comment farp and CAP sections, because of errors ( farp is not a variable of mission class ) and still another one, I don't remember.
 Could you explain what is wrong in the mission file, I would like to understand ( and make it work ) ?
Thank you for your quick answers and help.

**Comment by rp-** [comment]
Ok I found out whats wrong.
Luckily I had kinda the same issue once I did a fresh clone on my self from commandline and tried to run the script.

The problem is you have an older version of pydcs installed in python and as long you don't set the PYTHONPATH to the checkout version, python will import modules from the installed one first.
There are 2 ways to fix it:

- Uninstall the current installed version (`pip uninstall pydcs`), than everything should work from the git clone version
or
- Set the PYTHONPATH environment variable to the git clone of pydcs
  `set PYTHONPATH=C:/path/to/checkout`


The problem in the mission file was the incorrect serialization of the coalition enum in the mission triggers e.g.:
  `["coalitionlist"]=Coalition.Blue`
instead of
  `["coalitionlist"]="blue"`
